spec/classes/init_spec.rb

```diff
@@ -613,4 +613,34 @@
       })
     }
   end
+
+  context 'with parameter lookup_file_bind set' do
+    let :params do
+      { :lookup => ['file', 'bind'] }
+    end
+
+    it { should_not compile }
+  end
+
+  context 'with parameter lookup_file_bind set running on OpenBSD' do
```

Comment:
need a context that shows setting lookup but on a non-openbsd system and have it expect failure with your error message.